engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -420,7 +420,7 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
 
   onTransition {
     case _ -> toState if toState.terminal =>
-      workflowLogger.info(s"$tag done. Shutting down.")
+      workflowLogger.debug(s"$tag done. Shutting down WorkflowExecutionActor.")
```

Comment:
This has moved to the `debug` level, so normal users won't see it to be scared. They'll only turn on debugging, when requested, to send us a log so that we can debug a bug. In that case we can see the actor name already.

Parallel ToL: IMHO the "shutting down" is the scary part. Maybe comms team can come up better terminology if needed. I've seen them smooth over all sorts of UX issues.